datafusion/core/src/prelude.rs

```diff
@@ -36,5 +36,5 @@ pub use crate::logical_plan::{
     in_list, initcap, left, length, lit, lower, lpad, ltrim, max, md5, min, now,
     octet_length, random, regexp_match, regexp_replace, repeat, replace, reverse, right,
     rpad, rtrim, sha224, sha256, sha384, sha512, split_part, starts_with, strpos, substr,
-    sum, to_hex, translate, trim, upper, Column, JoinType, Partitioning,
+    sum, to_hex, translate, trim, upper, Column, Expr, JoinType, Partitioning,
```

Comment:
The lack of `Expr` in the prelude has always surprised me. I will make a separate PR for this

Reply:
https://github.com/apache/arrow-datafusion/issues/2347